Library/Formula/dcmtk.rb

```diff
@@ -1,8 +1,13 @@
 class Dcmtk < Formula
   homepage "http://dicom.offis.de/dcmtk.php.en"
+
+  # Current snapshot used for stable now.
   url "http://dicom.offis.de/download/dcmtk/snapshot/dcmtk-3.6.1_20150217.tar.gz"
   sha256 "3cf8f3e52ed8a5240a7facc3a118de411aa54bc9beccba0cf7a975735da35304"
   version "3.6.1-20150217"
+  revision 2
```

Comment:
Why the revision? Only necessary if it wasn't actually using our OpenSSL before. Should be `revision 1` if so.